docs/_docs/frontmatter.md

```diff
@@ -65,7 +65,8 @@ front matter of a page or post.
 
           If set, this specifies the layout file to use. Use the layout file
           name without the file extension. Layout files must be placed in the
-          <code>_layouts</code> directory.
+          <code>_layouts</code> directory. Set this to <code>null</code> if you
+          don't want to use any layout file.
```

Comment:
The downside to the proposition is that `layout: null` will be overridden by `frontmatter_defaults` defined in for the document (?) in the config file. As of Jekyll 3.5.x, this is overcome by setting `layout: none` in the document. (Works only for `Jekyll:Document` objects. For regular `Jekyll::Page` objects (like `./about.md`), there's a proposed port via an open PR.)

Reply:
So, currently to prevent override by `defaults` in `_config.yml`, you have to use?

- `layout: none` for documents
- `layout: null` for pages

Seems a bit messy.

Reply:
nope.. setting `layout: none` means it will not be overridden by `_config.yml`

Reply:
I know. That's why I said currently you have to use `layout: null` for pages. If you use `layout: none` for pages it'll try to use a layout named `none`.

So to summarize

- Using `layout: null` will produce a file without layout file, but this will be overridden if the file is a Document and has a layout defined in front matter defaults.
- (version 3.5.0 and above) Using `layout: none` in a Document produce a file without a layout file. If used in a Page, it will try to use a layout named `none`.

Reply:
yes, that is 99.99% correct.. :smiley:  If things go as I hope, `v3.6.0` might land with `layout: none` applicable to Pages as well..